deployments/kubernetes/cni-daemonset/istio-cni-daemonset.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: istio-cni-daemonset
+  namespace: istio-system
+  labels:
+    k8s-app: istio-cni-daemonset
+spec:
+  selector:
+    matchLabels:
+      k8s-app: istio-cni-daemonset
+  template:
+    metadata:
+      labels:
+        k8s-app: istio-cni-daemonset
+    spec:
+      serviceAccountName: istio-cni-daemonset-sa
+      tolerations:
+      # Tolerate all taints; we want to run on all nodes
+      - operator: "Exists"
+      containers:
+      - name: istio-cni-daemonset
+        image: gcr.io/istio-release:master-latest-daily
```

Comment:
this is not valid and master-latest-daily is dead

Reply:
Fixed the invalidity (when moving into other daemonset), but from what I can tell no alternate tags are being built. The CNI build pipeline doesn't appear to have run for a month or so. Going to leave it on `master-latest-daily` until I know what to change it to.